package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-in-angularjs",
-  "version": "17.7.0",
+  "version": "18.0.0",
```

Comment:
What do we do for users using older versions? Could just match whatever major versions the library supports similar to ReactDOM?

Reply:
It is not possible to support all previous versions of React
Upgrading to React 18 does not require a change in existing code, only in React's initialization method. All 10 forks of this library are aimed at upgrading to React 18. Also, most of the libraries are moving to React 18. 

In my opinion, it is worth upgrading the version of this library to 18.0.0 with React 18 support, and users who do not want to upgrade React can stay on 17.7 .1. The situation is the same in other libraries, either updating or saving the existing version.